docs/api/node.md

```diff
@@ -132,7 +132,17 @@ Starts an OTA firmware update process for this node. This method takes two argum
 -   `data` - A buffer containing the firmware image in a format supported by the device
 -   `target` - _(optional)_ The firmware target (i.e. chip) to upgrade. 0 updates the Z-Wave chip, >=1 updates others if they exist
 
-The library includes a helper method (exported from `zwave-js/Utils`) to extract the raw firmware image from some common file types:
+The library includes 2 helper method (exported from `zwave-js/Utils`) to guess the firmware format and extract the raw firmware image from some common file types:
```

Comment:
I know I originally wrote this, but we should mention which input files we support.

Reply:
It's mentioned later on extractFirmware function